recipes/maxbin2/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+set -euo pipefail
+cd src && make
```

Comment:
I modeled cd src && make after the docker build file. I like your idea of copying the executable to the right location.